official/resnet/estimator_cifar_benchmark.py

```diff
@@ -29,22 +29,33 @@
 from official.resnet import cifar10_main as cifar_main
 from official.utils.logs import hooks
 
-DATA_DIR = '/data/cifar10_data/cifar-10-batches-bin'
-
 
 class EstimatorCifar10BenchmarkTests(tf.test.Benchmark):
   """Benchmarks and accuracy tests for Estimator ResNet56."""
 
   local_flags = None
 
-  def __init__(self, output_dir=None):
+  def __init__(self, output_dir=None, root_data_dir=None):
+    """A benchmark class.
+
+    Args:
+      output_dir: directory where to output e.g. log files
+      root_data_dir: directory under which to look for dataset
+    """
+
     self.output_dir = output_dir
 
+    if root_data_dir is None:
```

Comment:
We will have to figure out how this will work in CNS.  I think it will be fine to move data around to match the pattern if we don't this will be very hard and maybe impossible to manage.  Not action needed.